locales/en/fields.json

```diff
@@ -135,5 +135,11 @@
     "heading": "Only include documents that will help plan the move.",
     "label": "Upload a file (optional)",
     "hint": "You can upload Word, Excel, PDF and JPEG documents up to 50 megabytes"
+  },
+  "move_agreed": {
+    "label": "Has this move been agreed with receiving prison?"
```

Comment:
What about "Move agreement status" as this label is then used for validation and I think that would read better.

So we would have

```
Move agreement status cannot be blank
```

instead of 

```
Has this move been agreed with receiving prison? cannot be blank
```